components/raftstore-v2/src/bootstrap.rs

```diff
@@ -33,8 +33,8 @@ pub struct Bootstrap<'a, ER: RaftEngine> {
     logger: Logger,
 }
 
-// Although all methods won't change internal state, but they still receive `&mut self` as it's
-// not thread safe to bootstrap concurrently.
+// Although all methods won't change internal state, but they still receive
```

Comment:
Maybe sketch the process of bootstrap here?

Reply:
Honestly I don't know what is "first region", could you help clarify?

Reply:
the initial region whose start key and end key are both empty to cover the whole keyspace range. Other regions would be split from this region after performing some writes